backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchBackendFactory.java

```diff
@@ -154,6 +160,8 @@ public BackendImplementor create(String name, BackendBuildContext buildContext,
 
 			indexLayoutStrategyHolder = createIndexLayoutStrategy( buildContext, propertySource );
 
+			Integer scrollTimeout = SCROLL_TIMEOUT.get( propertySource );
```

Comment:
By checking out this property when the backend is created, you're effectively making it a build-time property in Quarkus. I think it would make sense for users to change this property at deployment time, so I'd recommend checking it out when the backend starts, instead.

Currently all such "deployment-time" properties are related to the client (hosts, connection timeout, etc.). I think the easiest way to introduce the scroll timeout as a "deployment-time" property would be to move it to `ElasticsearchLinkImpl`. You can initialize it in `onStart` and add a getter similar to the other ones we already have there.